cmake/CMakeLists.txt

```
@@ -379,6 +379,14 @@ foreach(PKG_WITH_INCL KSPACE PYTHON VORONOI USER-COLVARS USER-MOLFILE USER-NETCD
   endif()
 endforeach()
 
+# optionally enable building script wrappers using swig
+option(WITH_SWIG "Build scripting language wrappers with SWIG" OFF)
+if(WITH_SWIG)
+  get_filename_component(LAMMPS_SWIG_DIR ${LAMMPS_SOURCE_DIR}/../tools/swig ABSOLUTE)
+  get_filename_component(LAMMPS_SWIG_BIN ${CMAKE_BINARY_DIR}/swig ABSOLUTE)
+  add_subdirectory(${LAMMPS_SWIG_DIR} ${LAMMPS_SWIG_BIN})
```

You are only using `LAMMPS_SWIG_BIN` once, also it doesn't need to be absolute.
```suggestion
  add_subdirectory(${LAMMPS_SWIG_DIR} swig)
```

---

changed. i was following the "better safe than sorry" guideline